### PR TITLE
done

### PR DESCRIPTION
**Pull request by anna-chovnova**
- [DEMO LINK](https://anna-chovnova.github.io/layout_stars/)
- [TEST REPORT LINK](https://anna-chovnova.github.io/layout_stars/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -8,6 +8,58 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars--0 container" >
+      <div class="star---image"></div>
```
Comment (ivan-kadykalo) [inline src/index.html:12]: According to name convention, use __ or - or -- to split words. Don't use --- in any place

File: src/index.html [hunk 2]
```diff
@@ -8,6 +8,58 @@
...
+      <div class="star---image"></div>
+      <div class="star---image"></div>
+      <div class="star---image"></div>
+      <div class="star---image"></div>
+    </div>
+
+    <div class="stars--1 container">
```
Comment (ivan-kadykalo) [inline src/index.html:19]:
```suggestion
    <div class="stars--1 stars">
```

File: src/index.html [hunk 3]
```diff
@@ -8,6 +8,58 @@
...
+      <div class="star---image line---001"></div>
```
Comment (ivan-kadykalo) [inline src/index.html:20]:
```suggestion
      <div class="star-image"></div>
```
Do you really need this modificator line-001? You can use your container name for this task

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+  height: 16px;
+  background-repeat: no-repeat;
+}
+
+.line---005:nth-child(-n+5) {
+  background-image: url(./images/star-active.svg);
+  margin-right: 4px;
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
```
Comment (ivan-kadykalo) [inline src/style.css:56]: DRY https://en.wikipedia.org/wiki/Don't_repeat_yourself

**Review by ivan-kadykalo — CHANGES_REQUESTED** [review]
Nice start 🤘

File: src/index.html (hunk 1 above)
Comment (ivan-kadykalo) [reply src/index.html:12]: Read the checklist again

File: src/index.html (hunk 3 above)
Comment (anna-chovnova) [reply src/index.html:20]: спробувала, але так в мене не працює

**Review by Ihor-Karpyn — APPROVED** [review]

**Pull request closed without merging**